cmd/snapctl/task.go

```diff
@@ -477,6 +480,10 @@ func listTask(ctx *cli.Context) error {
 		"LAST FAILURE",
 	)
 	for _, task := range tasks.ScheduledTasks {
+		lastErrorMessage := task.LastFailureMessage
+		if !ctx.Bool("verbose") && len(lastErrorMessage)+153 > termWidth && termWidth > 164 {
```

Comment:
I think adding a comment to say what they mean could be good though so that in the future if the format of the table changes we will know to update this also.

It also seems to work if the task name is shorter than the default one which was my other concern here.